CHANGES.txt

```diff
@@ -13,6 +13,14 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
 
         - Whatever John Doe did.
 
+  From Mats Wichmann:
+    - Tweak definition of default site_scons paths on Windows to conform
```

Comment:
Please list out the changes explicitly here and in RELEASE.txt.

Reply:
Added the new path to RELEASE.txt, and clarified the description in CHANGES.txt.